src/engine/runner/shared/npm.ts

```diff
@@ -0,0 +1,12 @@
+import { execSync } from "child_process";
+
+export const removeDependencies = (dirname: string, names: string[]) => {
+  const command = `npm uninstall ${names.join(" ")}`;
+
+  // eslint-disable-next-line no-console
```

Comment:
Lets go with winston

Reply:
IMO doesn't see a reason to use Winston to print output to the user. I think I need to set up the project without warning about using `console.log`